app/scripts/modules/core/cluster/cluster.service.js

```diff
@@ -66,10 +66,8 @@ module.exports = angular.module('spinnaker.core.cluster.service', [
         outOfService: 0,
         total: 0,
       };
-      if (!cluster.serverGroups) {
-        return;
-      }
-      cluster.serverGroups.forEach(function(serverGroup) {
+      var operand = cluster.serverGroups || cluster.jobs;
```

Comment:
Just to play it safe, we should probably change this to:
```javascript
var operand = cluster.serverGroups || cluster.jobs || [];
```
I'm not sure what scenario led to the old code being there, but...let's assume it was there for a reason.